### DISCUSSION TITLE
How to disable the swipe event on a specific element?

### DISCUSSION THREAD

**Discussion by hugomendonca98** [igordanchenko/yet-another-react-lightbox#184] [Q&A] [answered]
Hello, I'm using the vimejs video player, because I need it to play videos in hsl format, but I'm having a small problem, when I try to click and drag the video progress bar, the slide swipe event is triggered:
![image](https://github.com/igordanchenko/yet-another-react-lightbox/assets/47926683/ab58dca3-3129-429d-b9e1-c4fb4be1e96b)

**Comment by igordanchenko** [comment, marked as answer]
Can you see if cancelling propagation of the onPointerDown event addresses this issue? You can add onPointerDown event handler to any element wrapping your video player.

**Reply by hugomendonca98** [reply]
It worked @igordanchenko, thank you very much!
